templates/docs/examples/base/playground.html

```diff
@@ -0,0 +1,17 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Links{% endblock %}
+
+{% block standalone_css %}base{% endblock %}
+
+{% block content %}
```

Comment:
Was this intended to be part of this PR?

Reply:
Nope, removed